t/additional_filters.t

```diff
@@ -0,0 +1,53 @@
+use Test::Nginx::Socket 'no_plan';
+use Cwd qw(cwd);
+
+my $pwd = cwd();
+
+our $HttpConfig = qq{
+    lua_package_path "$pwd/lib/?.lua;;;";
+    lua_package_cpath "/usr/local/openresty/lualib/?.so;;";
+};
+
+run_tests();
+
+__DATA__
+
+=== TEST 1: get filter.
+--- http_config eval: $::HttpConfig
+--- config
+    location /t {
+        content_by_lua_block {
+            local context = {}
+            context.foo = "123"
+
+            local Liquid = require 'liquid'
+            local document = 'SELF::{{ self | first | get: "foo" }}'
```

Comment:
Why is `|  first` needed?

Reply:
the context is added by table.insert, so it's an array, instead of a normal map.  

https://github.com/3scale/liquid-lua/blob/master/lib/liquid.lua#L2497